### ISSUE TITLE
Microseconds to be taken into account

### ISSUE THREAD

**Issue by AserMata** [pmlrsg/PySpectra#15]
Spectrometer can take several measures per second, similar with the GPS in the drone. To make the processing more robust I suggest to use the microsecond information as well (enough to remove the int).

https://github.com/pmlrsg/PySpectra/blob/a17c536e02c93f73e897f35172c84a676a0c0fba/PySpectra/ocean_optics.py#L124

**Comment by danclewley** [comment]
Sounds sensible, do you want to open a pull request for the changes?

**Comment by AserMata** [comment]
Yes, I will do later. After further testing I realised that both windows and raspberry pi were saving only with second accuracy so this feature is still a good idea but not a pressing matter.
